client/src/pages/home/home.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import logo from '../../logo.svg';
+import './home.css'; //import css for this page
+function Home() {   {/* This page is the exact same page as the original example */}
+  return (
+    <div className="Home">
```

Comment:
You should replace this div with a `main` tag, follow semantic html for best accessibility practices

Reply:
Yep! I changed most if not all the `<div>` statements to more descriptive semantic HTML5 format.